pixel-point/pixelpoint-website
# chore: add Alexey Tretyakov as author and replace Vlad Kamelsky to Ale…

**This PR brings the following changes**
 adds Alexey Tretyakov as an author and replace Vlad Kamelsky to Alexey Tretyakov in Blog articles (for a while).

**Steps to test**

1. Open the [page](https://pixelpointwebsite-authorupdate.gtsb.io/blog/)
2. Confirm that everything looks and works as expected

**Screenshots**
<img width="2040" alt="Screenshot 2022-08-25 at 09 07 45" src="https://user-images.githubusercontent.com/75138180/186572740-9342001b-5312-410d-a5ea-764e26068776.png">
